pkgs/applications/altcoins/ledger-live-desktop/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+ stdenv,
+ appimage-run,
+ fetchurl,
+ runtimeShell,
+}:
+
+stdenv.mkDerivation rec {
+  github-user = "LedgerHQ";
+  pname = "ledger-live-desktop";
+  version = "1.9.1";
+
+  src = fetchurl {
+    url = "https://github.com/${github-user}/${pname}/releases/download/v${version}/${pname}-${version}-linux-x86_64.AppImage";
+    sha256 = "0b919ilvv3zi17fnzngnnmg28dxqlq0dvj3fdvb50kh3ibrhdcfz";
+  };
+
+  buildInputs = [ appimage-run];
+
+  unpackPhase = ":";
+
+  installPhase = ''
+    mkdir -p $out/{bin,share}
+    cp $src $out/share/${pname}
+    echo "#!${runtimeShell}" > $out/bin/${pname}
+    echo "${appimage-run}/bin/appimage-run $out/share/${pname}" >> $out/bin/${pname}
+    chmod +x $out/bin/${pname} $out/share/${pname}
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Ledger Live (Desktop)";
```

Comment:
The description shouldn't really be the name of the package. I found "The companion to your Ledger hardware wallet" on their website but that's more of a slogan so there is almost certainly something better.

Reply:
@alexarice this is the description pasted from their github repo

Reply:
meta.description should:

    Be capitalized.
    Not start with the package name.
    Not have a period at the end.

This is what contributing.md says and it doesn't seem like a helpful description, even if it is the description on their github